### PR TITLE
Add locking and skipping of the review

### PR DESCRIPTION
**Pull request by tm-kn**
#24 #55

### CODE REVIEW

File: mresponse/reviews/api/serializers.py [hunk 1]
```diff
@@ -35,3 +37,9 @@ def get_response_url(self, instance):
             kwargs={'review_pk': instance.pk},
             request=self.context.get('request'),
         )
+
+    def get_skip_url(self, instance):
```
Comment (kaedroho) [inline mresponse/reviews/api/serializers.py]: This URL isn't specific to this review. Also, it probably shouldn't be shown if this review is not assigned to the current user.

File: mresponse/reviews/models.py [hunk 2]
```diff
@@ -32,6 +33,14 @@ class Review(models.Model):
     review_text = models.TextField()
     review_rating = models.SmallIntegerField(choices=REVIEW_RATING_CHOICES)
     last_modified = models.DateTimeField()
+    assigned_to = models.OneToOneField(
```
[the remark this answers is not in the document]
Comment (tm-kn) [reply mresponse/reviews/models.py]: Yes. At least for now.

File: mresponse/reviews/api/serializers.py (hunk 1 above)
Comment (tm-kn) [reply mresponse/reviews/api/serializers.py]: This endpoint will be shown only to the user who is writing the response, therefore we don't have to worry about it I think. We can create two separate views if we ever need "public" view.

Comment (tm-kn) [reply mresponse/reviews/api/serializers.py]: Also, I've added the ID to the URL so the client does not try to skip outdated reviews.

Comment (kaedroho) [reply mresponse/reviews/api/serializers.py]: Thanks, looks good to me now!

**Pull request merged**